views/home.hbs

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Firefox Breach Alerts</title>
+        <link rel="stylesheet" href="/css/foundation.min.css">
+        <script type="text/javascript" src="/js/app.js" async></script>
+        <script type="text/javascript" src="/js/vendor/jquery.js" defer></script>
+        <script type="text/javascript" src="/js/vendor/foundation.min.js" defer></script>
```

Comment:
(a) Should these `<script ...>` tags be at the bottom of `<body>` so they aren't blocking?
(b) Ridiculous pre-optimization question, but should we be including Twitter/OpenGraph `<meta>` tags (tl;dr: should i file a separate bug to add those)?

Reply:
(a) `async` and `defer` attributes on script tags are the new hotness to gain the performance of async & deferred script loading.

(b) yeah, follow-up.

Reply:
Filed; #54